src/Parser/Paladin/Retribution/Modules/Talents/RighteousVerdict.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS';
+import SpellLink from 'common/SpellLink';
+import SpellIcon from 'common/SpellIcon';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+import { formatNumber, formatPercentage } from 'common/format';
+
+import Analyzer from 'Parser/Core/Analyzer';
+
+import GetDamageBonus from 'Parser/Paladin/Shared/Modules/GetDamageBonus';
+
+const RIGHTEOUS_VERDICT_MODIFIER = 0.15;
+
+class RighteousVerdict extends Analyzer {
+  damageDone = 0;
+  spenderInsideBuff = 0;
+  totalSpender = 0;
+
+  constructor(...args) {
+    super(...args);
+    this.active = this.selectedCombatant.hasTalent(SPELLS.RIGHTEOUS_VERDICT_TALENT.id);
+  }
+
+  on_byPlayer_cast(event) {
+    const spellId = event.ability.guid;
+    if (spellId === SPELLS.TEMPLARS_VERDICT.id) {
+      if (this.selectedCombatant.hasBuff(SPELLS.RIGHTEOUS_VERDICT_BUFF.id)) {
+        this.spenderInsideBuff++;
+      }
+      this.totalSpender++;
+    }
+  }
+
+  on_byPlayer_damage(event) {
+    const spellId = event.ability.guid;
+    if (!this.selectedCombatant.hasBuff(SPELLS.RIGHTEOUS_VERDICT_BUFF.id)) {
```

Comment:
I'd suggest adding a buffer here, might be why you're seeing 0 buffed casts in the logs.